### DISCUSSION TITLE
List of components hint

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#18392] [Questions] [answered]
Is there any way to find out the list of all available components that have been autoloaded in Nuxt? It would be nice to have some kind of hint in vscode. Is there something like that?

**Comment by MorevM** [comment, marked as answer]
Are you looking for `.nuxt/components.d.ts` file?
VSCode shows them in template section.

![autocomplete of components in VSCode](https://user-images.githubusercontent.com/49679666/199966697-e0876644-a43d-4032-a498-d5dbb14bf599.jpeg)

It's not separated from other global things though, but I think it's more in the scope of Volar..

**Reply by theguriev** [reply]
Hm... Yes. I guess you're right. It's more about the Volar. Thank you 👍
